lib/iphone-simulator-xcode-simctl.ts

```diff
@@ -178,6 +181,16 @@ export class XCodeSimctlSimulator extends IPhoneSimulatorNameGetter implements I
 		}
 	}
 
+	private verifyDevice(device?: IDevice): void {
+		const availableDevices = this.getDevices();
+
+		if(_.find(availableDevices, { id: device.id })){
```

Comment:
I would invert the `if`:
```TypeScript
if(!_.find(availableDevices, { id: device.id })) {
    errors.fail(`No simulator image available for device identifier '${device.id}'.`);
}
```
This way you can skip the `return` inside the method.